pkg/controllers/organization/teamrole_seeder_controller.go

```diff
@@ -0,0 +1,93 @@
+// SPDX-FileCopyrightText: 2024 SAP SE or an SAP affiliate company and Greenhouse contributors
+// SPDX-License-Identifier: Apache-2.0
+
+package organization
+
+import (
+	"context"
+
+	corev1 "k8s.io/api/core/v1"
+	rbacv1 "k8s.io/api/rbac/v1"
+	"k8s.io/client-go/tools/record"
+	ctrl "sigs.k8s.io/controller-runtime"
+	"sigs.k8s.io/controller-runtime/pkg/client"
+	"sigs.k8s.io/controller-runtime/pkg/controller/controllerutil"
+	"sigs.k8s.io/controller-runtime/pkg/log"
+
+	greenhousesapv1alpha1 "github.com/cloudoperators/greenhouse/pkg/apis/greenhouse/v1alpha1"
+	"github.com/cloudoperators/greenhouse/pkg/clientutil"
+)
+
+var defaultTeamRoles = map[string]greenhousesapv1alpha1.TeamRoleSpec{
+	"cluster-admin": {
+		Rules: []rbacv1.PolicyRule{{
+			APIGroups: []string{"*"},
+			Resources: []string{"*"},
+			Verbs:     []string{"*"},
+		}},
+	},
+}
+
+// TeamRoleSeederReconciler reconciles a Organization object
+type TeamRoleSeederReconciler struct {
+	client.Client
+	recorder record.EventRecorder
+}
+
+//+kubebuilder:rbac:groups=greenhouse.sap,resources=organizations,verbs=get;list;watch;create;update;patch;delete
+//+kubebuilder:rbac:groups=greenhouse.sap,resources=organizations/status,verbs=get;update;patch
+//+kubebuilder:rbac:groups=greenhouse.sap,resources=organizations/finalizers,verbs=update
```

Comment:
This controller does not need all the permissions.
```suggestion
//+kubebuilder:rbac:groups=greenhouse.sap,resources=organizations,verbs=get;list;watch
```